core/lib/CMakeLists.txt

```diff
@@ -14,11 +14,10 @@ add_subdirectory(snappy)
 add_subdirectory(leveldb)
 
 add_subdirectory(spdlog)
-if (SQLCIPHER)
-    include(ProjectSQLiteCipher)
-else()
-    add_subdirectory(sqlite3)
-endif()
+add_subdirectory(sqlcipher)
```

Comment:
Ditto.

Reply:
no more sqlite3 - we have other implementation of this functionality in sqlcipher.